kornia/color/colormap_data.py

```diff
@@ -0,0 +1,1263 @@
+from typing import List
+
+# All colormaps has 64 base colors. Extracted from matplotlib.
+# https://matplotlib.org/stable/users/explain/colors/colormaps.html
+
+
+RGBColor = List[float]
+
+
+_AUTUMN_BASE: List[RGBColor] = [
```

Comment:
also create functions to return each cmap base, this way these lists aren't created on import time

Reply:
I'm not sure what do you mean. Could you give me an example?

Reply:
Instead of declaring it in the body of the file, put each one of these lists inside a getter function this way, the lists will be "created" just when we call the function, otherwise, python will "create"/"allocate" this lists when we import this file... 
```suggestion
def get_autumn_base():
    return [
    ...
    ]
```